HopsanGUI/GUIObjects/GUIModelObject.cpp

```diff
@@ -636,6 +636,17 @@ bool ModelObject::isDisabled()
     }
 }
 
+bool ModelObject::isAnimationDisabled()
+{
+    // Does nothing by default, should be overloaded
```

Comment:
I am not sure, but a return may be needed here to prevent warning ?

Maybe the function (and others) should be pure virtual. I dont think that we will ever have an instance of ModelObject directly

Reply:
I removed it from base class completely. It is only relevant for container objects anyway.